data/role/jenkins-slave-android.yaml

```diff
@@ -0,0 +1,65 @@
+---
+psick::base::linux_classes:
+  packages: psick::packages
+  andoid: profile::android
+  nodejs: profile::nodejs  
+  ssh: ''
+
+psick::profiles::linux_classes:
+  ssh: 'psick::openssh'
+
+
+psick::packages::packages_list:
+  - awscli
+  - ncdu
+  - unzip
+  - gradle
+  - xmlstarlet
+
+profile::nodejs::version: 10.x
+profile::android::sdk_tools: sdk-tools-linux-4333796.zip
+profile::android::sdk_path: /home/ubuntu
+
+profile::android::npm_packages:
+  - "ionic"
+  - "cordova"
+
+profile::android::sdk_packages:
+  - "platform-tools"
+  - "system-images;android-16;google_apis;x86"
```

Comment:
@msmadja Do we need all of these versions to build the Android app? (See lines 29-39.)